README.md

```diff
@@ -236,3 +234,47 @@ $ chembl_downloader --test
+|              3   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|              4   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|              5   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|              6   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|              7   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|              8   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|              9   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             10   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             11   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             12   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             13   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             14   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             15   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             16   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             17   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             18   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             19   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             20   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             21   | Yes                  |                                                                                                 |
+|             22   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
```

Comment:
Need to add `/archived` like in https://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/releases/chembl_22/archived/

Reply:
Do you want to fix this code for working for other versions? I thought you mentioned you don't want to in the earlier comments.

Reply:
To be clear, I explicitly have no interest in supporting the MySQL nor Oracle dumps of the ChEMBL database.

Reply:
Anyway this fix was pushed in https://github.com/cthoyt/chembl-downloader/commit/fffd13cee030cf1bcf41136f58ab1138391bccaf and I already updated this branch with it